ci/windows/test_py2_cpu.ps1

```diff
@@ -16,11 +16,18 @@
 # under the License.
 
 7z x -y windows_package.7z
+function New-TemporaryDirectory {
+    $parent = [System.IO.Path]::GetTempPath()
+    [string] $name = [System.Guid]::NewGuid()
+    New-Item -ItemType Directory -Path (Join-Path $parent $name)
+}
 $env:MXNET_LIBRARY_PATH=join-path $pwd.Path windows_package\lib\libmxnet.dll
 $env:PYTHONPATH=join-path $pwd.Path windows_package\python
 $env:MXNET_STORAGE_FALLBACK_LOG_VERBOSE=0
+$env:MXNET_HOME=New-TemporaryDirectory
```

Comment:
This is what I wrote up to solve this issue
```
c:\Anaconda3\envs\py2\Scripts\pip show virtualenv
if ($LASTEXITCODE == 1) {
    c:\Anaconda3\envs\py2\Scripts\pip install virtualenv
    c:\Anaconda3\envs\py2\Scripts\pip install virtualenvwrapper-win
}
$parent = [System.IO.Path]::GetTempPath()
[string] $name = [System.Guid]::NewGuid()
New-Item -ItemType Directory -Path (Join-Path $parent $name)
cd $name
New-VirtualEnvironment virtual_test_env
```

Reply:
Please give me a quick heads up when you changed the path to a workspace relative one